### ISSUE TITLE
content.googleapis.com not loaded 400 ()

### ISSUE THREAD

**Issue by Jagdeep-Sing** [gsuitedevs/browser-samples#15]
This is the error i got when follows the javascript quick start guide :

<img src="https://i.imgur.com/LV85OS2.png">

Please guide to resolve this.
Thanks.

**Comment by erickoledadevrel** [comment]
This error indicates that the `CLIENT_ID` and `API_KEY` variables in the code haven't been set correctly. Can you double check that you've pasted the correct values into both variables?

**Comment by lpender** [comment]
Hi,

I'm having the same issue.

I have a key that was auto-named "OAuth Key" when I generated it by following the steps here:

https://developers.google.com/identity/sign-in/web/sign-in

It worked for Sign In.

I ran the wizard again, following the instructions here:

https://developers.google.com/drive/api/v3/quickstart/js

It said I already had the credentials necessary and listed the "OAuth Key" and the "Web Client (Auto-created for Google Sign-in)".

Both keys were generated, though it's not clear which I should be plugging in in this context.

Both are returning 400 errors.

It's also not clear to me whether I should be including the `.apps.googleusercontent.com ` at the end of the Client ID.

Also, inconsistent naming between "Client secret" and "API_KEY", which I assume are the same? 

Thanks!
-Lee
